packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -320,21 +290,72 @@ $heading-xsmall-text: (
         font-size: 1.25rem,
         font-style: normal,
         font-weight: 300,
-        line-height: 1.4,
+        line-height: 1.3,
     ),
     tablet: (
         font-family: "'Ebony', sans-serif",
         font-size: 1.25rem,
         font-style: normal,
         font-weight: 300,
-        line-height: 1.4,
+        line-height: 1.3,
     ),
     desktop: (
         font-family: "'Ebony', sans-serif",
         font-size: 1.25rem,
         font-style: normal,
         font-weight: 300,
-        line-height: 1.4,
+        line-height: 1.3,
+    ),
+) !default;
+
+$link: (
```

Comment:
Please set up all typography in Figma. These tokens are from supernova and they are missing some info which is automatically outputted from Figma. I assume you need the style in Figma anyway.

In Supernova you can tell if the token is from Figma or Supernova by the icon in the value column 
<img width="245" alt="image" src="https://github.com/lmc-eu/jobs-design-system/assets/7041807/d89ac7f5-d4cc-4077-8231-5b346a231595">